docs/installingNative.md

```diff
@@ -52,6 +52,9 @@ python3.6 --version   # should return "Python 3.6" or higher
 ```sh
 cd PrairieLearn/images/plbase
 python3 -m pip install -r python-requirements.txt
+
+# Alternatively, use the following command to transpile packages and update Node/Python libraries:
+make deps
```

Comment:
> I'd rather restructure this so that this is the first recommended things, and the individual steps are listed as a cohesive "otherwise do this" option. I can push what I have in mind if you'd like.

I'd appreciate your suggestion here, if you have the time. Otherwise I'll suggest something when I'm back in front of a computer.

Reply:
ebd4f70 - let me know if that looks good to you.

Reply:
These look good, thanks.